preprocessors/openstreetmap/main.py

```diff
@@ -54,26 +65,36 @@ def get_map_data():
 
     if validated is not None:
         return validated
-
+    time_stamp = int(get_timestamp())
+    name = "ca.mcgill.a11y.image.preprocessor.openstreetmap"
+    request_uuid = content["request_uuid"]
     # Check if this request is for an openstreetmap
     if 'coordinates' not in content and 'placeID' not in content:
-        logging.info("Not map content. Skipping...")
-        return "", 204
+        LOGGER.info("Not map content. Skipping...")
+        return jsonify(""), 204
 
     # Build OpenStreetMap request
     coords = get_coordinates(content)
     if coords is None:
-        error = 'Unable to find Latitude/Longitude'
-        logging.error(error)
-        return jsonify(error), 400
+        error = 'Unable to find latitude/longitude'
+        LOGGER.error(error)
+        return jsonify(""), 400
 
     latitude = coords["latitude"]
     longitude = coords["longitude"]
+
+    if latitude is None:
```

Comment:
If `coordinates` is present, and you validate the request, this will never actually come up.

Reply:
As a matter of fact, this as well as the logic in ```line 78```  of the ```main.py``` is all redundant and shouldn't be there anymore. Thanks for pointing it out.